index.html

```diff
@@ -134,13 +133,18 @@ <h3>Milestones</h3>
 
             $("#addrow").on("click", function () {
                 var newRow = $("<tr>");
-                var cols = "";
-
-                cols += '<td><input type="text" class="form-control" name="name' + counter + '" placeholder="name"/></td>';
-                cols += '<td><input type="text" class="form-control" name="url' + counter + '" placeholder="url"/></td>';
-                cols += '<td><input type="text" class="form-control" name="icon' + counter + '" placeholder="icon"/></td>';
-
-                cols += '<td><buntton class="btn btn-sm btn-danger ibtnDel">Delete</buntton></td>';
+                var cols = `
+                <td>
+                    <input type="text" class="form-control" placeholder="Text Link" name="name${counter}" required/>
+                </td>
+                <td>
+                    <input type="text" class="form-control" placeholder="Link URL" name="url${counter}" required/>
+                </td>
+                <td>
+                    <input type="text" name="icon${counter}" class="form-control" placeholder="icon" />
+                </td>
+                <td><buntton class="btn btn-sm btn-danger ibtnDel">Delete</buntton></td>
```

Comment:
There appears to be a typo in the button tag
```suggestion
                <td><button class="btn btn-sm btn-danger ibtnDel">Delete</button></td>
```

and it should also have type="button" because it is not a submit button

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix